scripts/lib/sphinx/generateToc.ts

```diff
@@ -125,6 +126,11 @@ export function generateToc(options: {
     tocChildren.push(...orderEntriesByTitle(nestedTocModules));
   }
 
+  tocChildren.push({
+    title: "Release notes",
+    url: pkg.releaseNotesUrl,
+  });
```

Comment:
For Qiskit, this needs to be a folder with one file per minor release. I recommend not dealing with Qiskit in this PR yet. Get Runtime and Provider working well since they're much simpler.

For example, Qiskit is complex because we need to split up the legacy HTML page into several pages ourselves, and we need to remove all the non-Terra stuff. That is a one-time migration and probably shouldn't be in this script.